LethalAPI.Core/LethalAPI.Core.csproj

```diff
@@ -13,48 +13,52 @@
             https://api.nuget.org/v3/index.json;
             https://nuget.bepinex.dev/v3/index.json
         </RestoreAdditionalProjectSources>
-        <DocumentationFile>..\bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
-        <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)LethalAPI.ruleset</CodeAnalysisRuleSet>
-        <OutputPath>..\bin\$(Configuration)\</OutputPath>
+        <DocumentationFile>..\bin\Debug\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+        <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\LethalAPI.ruleset</CodeAnalysisRuleSet>
+        <OutputPath>..\bin\Debug\</OutputPath>
         <Nullable>enable</Nullable>
+        <Platform>AnyCPU</Platform>
+        <Configurations>Release</Configurations>
+        <Platforms>AnyCPU</Platforms>
+        <Optimize>true</Optimize>
+        <DefineConstants>trace</DefineConstants>
     </PropertyGroup>
-
     <ItemGroup>
-        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all" ExcludeAssets="RUNTIME" />
-        <PackageReference Include="BepInEx.Core" Version="5.*" ExcludeAssets="RUNTIME" />
-        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" ExcludeAssets="RUNTIME" />
-        <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.507" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" ExcludeAssets="RUNTIME" />
-        <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" ExcludeAssets="RUNTIME" />
+        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all" ExcludeAssets="runtime" />
+        <PackageReference Include="BepInEx.AssemblyPublicizer.MSBuild" Version="0.4.1" PrivateAssets="all" ExcludeAssets="runtime" />
+        <PackageReference Include="BepInEx.Core" Version="5.*" ExcludeAssets="runtime" />
+        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" ExcludeAssets="runtime" />
+        <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.507" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" ExcludeAssets="runtime" />
+        <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" ExcludeAssets="runtime" />
+        <PackageReference Include="YamlDotNet" Version="13.7.1" ExcludeAssets="runtime" />
```

Comment:
Can you add the json lib reference while you're at it?

Reply:
Good idea.